baseapp_comments/graphql/object_types.py

```
@@ -138,31 +147,73 @@ def get_node(self, info, id):
             return None
         return node
 
+    def resolve_target(root, info, **kwargs):
+        if not root.target_document_id:
+            return None
+
+        request_cache = getattr(info.context, "_comment_target_cache", None)
+        if request_cache is None:
+            request_cache = {}
+            setattr(info.context, "_comment_target_cache", request_cache)
```

Ruff (0.15.11)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)